enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -301,35 +313,189 @@ status:
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: ztp-placement
+  namespace: ztp-acm-ns
+spec:
+  predicates:
+    - requiredClusterSelector:
+        labelSelector:
+          matchExpressions:
+            - key: common-profile
+              operator: In
+              values:
+                - 'true'
+  DecisionStrategy:
+    groupStrategy:
+      clustersPerDecisionGroup: 100%
+      DecisionGroups:
+      - groupName: prod-canary
+        clusterSelector:
+          matchExpressions:
+            - key: prod-canary
+              operator: Exist
+
+status:
+  numberOfSelectedClusters: 320
+  decisionGroups:
+  - decisionGroupIndex: 0
+    placementDecisions:
+    - ztp-placement-decision-0
+    clusterCount: 20
+  - decisionGroupIndex: 1
+    placementDecisions:
+    - ztp-placement-decision-1
+    - ztp-placement-decision-2
+    - ztp-placement-decision-3
+    clusterCount: 300
+```
+Another example for a placement having decisionStrategy defined with clustersPerDecisionGroup=150 as below. With a selection out of 320 clusters there are 3 decisionGroup are created.
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: ztp-placement
+  namespace: ztp-acm-ns
+spec:
+  predicates:
+    - requiredClusterSelector:
+        labelSelector:
+          matchExpressions:
+            - key: common-profile
+              operator: In
+              values:
+                - 'true'
+  DecisionStrategy:
+    groupStrategy:
+      clustersPerDecisionGroup: 150
+status:
+  numberOfSelectedClusters: 320
+  decisionGroups:
+  - decisionGroupIndex: 0
+    placementDecisions:
+    - ztp-placement-decision-0
+    - ztp-placement-decision-1
+    clusterCount: 150
+  - decisionGroupIndex: 1
+    placementDecisions:
+    - ztp-placement-decision-2
+    - ztp-placement-decision-3
+    clusterCount: 150
+  - decisionGroupIndex: 2
+    placementDecisions:
+    - ztp-placement-decision-4
+    clusterCount: 20
+```
+
+The Placement Helper [library](https://github.com/open-cluster-management-io/api/blob/main/cluster/v1beta1/helpers.go) should provide functions to facilitate retrieve the decision groups.
 
 #### Special case handling
 
 * Clusters have been removed from decision group index 1 and decision group index 2 has been created. In order to not shift current selected clusters in the decision groups, placement controller will keep append clusters to decision group index 2 till it reach max number of clusters per decision group then append new cluster to decision group index 1 till it reach the max number of clusters per decision group.
 
+* The PrioritizerPolicy has been defined by end user and a new cluster has been added to the placement selection after the decision Groups are created. The clusters wil need to be adjusted within the decision groups based on the new cluster decision group index.
```

Comment:
Could here provide an example of how the group and index for the clusters will adjust? 

My understanding whether a placement defines PrioritizerPolicy or not, the clusters in the placement decision are ordered by name rather than score (this is the current behavior to ensure when the cluster score changes but selected clusters keep the same, the placement decision won't change), so the special case here is mainly about how to handle the newly added clusters.

Reply:
yes, it is about the newly added clusters. I'm not quit familiar with PrioritizerPolicy but mainly this section is to be sure at implementation we take care of this case and we don't break the behavior.

Reply:
For the added or deleted cluster, I think a better way is always to recalculate the groups after a change, to ensure 2 placements (1 placement created before cluster change, another placement created after cluster change) have the same result.  
As for the consumer (addon, policy, etc), they could record the groups and determine how to handle the added/deleted clusters.